src/makeRobot.js

```diff
@@ -38,7 +38,75 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward(number = 1) {
+      if (number <= 0) {
+        return this;
+      }
+
+      this.coords.y += number;
+
+      return this;
+    },
+
+    goBack(number = 1) {
+      if (number <= 0) {
+        return this;
+      }
+
+      this.coords.y -= number;
+
+      return this;
+    },
+
+    goRight(number = 1) {
+      if (number <= 0) {
+        return this;
+      }
+
+      this.coords.x += number;
+
+      return this;
+    },
+
+    goLeft(number = 1) {
+      if (number <= 0) {
+        return this;
+      }
+
+      this.coords.x -= number;
+
+      return this;
```

Comment:
```suggestion
      if (number > 0) {
         this.coords.x -= number;
      }
      return this;
```
that's little simpler I think 😄